src/scenes/PlantsDashboardRouter/components/ZoneLevelDataMap.tsx

```diff
@@ -213,15 +214,22 @@ export default function ZoneLevelDataMap({ plantingSiteId }: ZoneLevelDataMapPro
       (pz: ObservationPlantingZoneResultsWithLastObv) => {
         return {
           ...pz,
-          lastObv: lastZoneObservation(zoneObservations?.[pz.plantingZoneId])?.startDate,
-          plantingSubzones: pz.plantingSubzones.map((oldSubzone) => ({
-            ...oldSubzone,
-            lastObv: lastSubZoneObservation(
+          lastObv:
+            lastZoneObservation(zoneObservations?.[pz.plantingZoneId])?.completedTime ||
+            lastZoneObservation(zoneObservations?.[pz.plantingZoneId])?.startDate,
```

Comment:
This seems counterintuitive to me since we shouldn't be showing in-progress observation stats on the plants dashboard.

Reply:
Latest completed observation, right?  That's the distinction I'm making.  We shouldn't be showing in-progress data on the plants dashboard map.

Reply:
Yes, in the popup it's the latest completed observation. This part is used for recency and it was searching between all observations. Just added on more filter to get completed ones